docs/4-frameworks/03-Vue.md

```diff
@@ -34,18 +107,21 @@ Vue.config.ignoredElements = [/^ui5-/];
 - If using a build step: pass the `isCustomElement` option to the Vue template compiler. If using `vue-loader`, this should be passed via vue-loader's `compilerOptions` option:
 
 ```js
-// in webpack config
-rules: [
-	{
-		test: /\.vue$/,
-		use: 'vue-loader',
-		options: {
-			compilerOptions: {
-				isCustomElement: tag => tag.startsWith("ui5-")
-			}
-		}
-	}
-]
+// in vue.config file
```

Comment:
vue.config is not there when using vitejs, so delete this whole section

Reply:
sorry, I missed that this is a second section.

still, the official vue site getting started recommends
`npm init vue@latest`, which is creates a vite project as the documentation states:
> The created project will be using a build setup based on [Vite](https://vitejs.dev/) and allow us to use Vue [Single-File Components](https://vuejs.org/guide/scaling-up/sfc.html) (SFCs).

I suggest to leave creating a project to the official docs as they change too much and describe the web component steps here. In any case, the webpack config should be deleted as quite old.